packages/@aws-cdk/aws-wafv2/README.md

```diff
@@ -21,3 +21,20 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 ```ts
 import wafv2 = require('@aws-cdk/aws-wafv2');
 ```
+
+### IPSet
+
+Use an AWS::WAFv2::IPSet to identify web requests that originate from specific IP addresses or ranges of IP addresses. For example, if you're receiving a lot of requests from a ranges of IP addresses, you can configure AWS WAF to block them using an IP set that lists those IP addresses.
```

Comment:
Reference the CDK class instead of the CFN resource. Use backticks for formatting.